src/delimiter.js

```diff
@@ -453,7 +453,7 @@ const makeStackedDelim = function(
         Style.TEXT, options, classes);
 };
 
-// All surds have 0.08em padding above the viniculum inside the SVG.
+// All surds have 0.08em padding above the vinculum inside the SVG.
```

Comment:
This item is consistently spelled this way, but the world disagrees...

https://www.merriam-webster.com/dictionary/vinculum

this is a TI(re)L - Google Sheets told me.